pkg/dra/claims.go

```diff
@@ -33,38 +33,69 @@ import (
 )
 
 var (
-	ErrDeviceClassNotMapped = errors.New("DeviceClass is not mapped in DRA configuration")
-	ErrResourceClaimInUse   = errors.New("ResourceClaim is in use")
-	ErrClaimSpecNotFound    = errors.New("failed to get claim spec")
+	ErrDeviceClassNotMapped  = errors.New("DeviceClass is not mapped in DRA configuration")
+	ErrResourceClaimInUse    = errors.New("ResourceClaim is in use")
+	ErrClaimSpecNotFound     = errors.New("failed to get claim spec")
+	ErrUnsupportedDRAFeature = errors.New("unsupported DRA feature")
+
+	// Specific unsupported DRA feature errors
+	ErrUnsupportedDRADeviceConstraints = errors.New("device constraints (MatchAttribute) are not supported")
+	ErrUnsupportedDRADeviceConfig      = errors.New("device config is not supported")
+	ErrUnsupportedDRAFirstAvailable    = errors.New("FirstAvailable device selection is not supported")
+	ErrUnsupportedDRACELSelectors      = errors.New("CEL selectors are not supported")
+	ErrUnsupportedDRAAdminAccess       = errors.New("AdminAccess is not supported")
+	ErrUnsupportedDRAAllocationModeAll = errors.New("AllocationMode 'All' is not supported")
```

Comment:
Do these need to be exposed? If not then unexport (lowercase)

Reply:
+1

Reply:
@mimowo if the concern is support these variable as a public library because folks will start using these exported values, then an alternative would be to hide it behind an internal package such that the errors are only available to kueue